docs/robots/jackal/integration_jackal.mdx

```diff
@@ -1,12 +1,98 @@
 ---
-title: Integration
+title: Jackal Integration
```

Comment:
This should just be 'Integration' to be aligned with our other robots.

Reply:
Or, add a 'sidebar_label' line to the YAML header.

---
title: Jackal Integration
sidebar_label: Integration
sidebar_position: 2
toc_min_heading_level: 2
toc_max_heading_level: 4
---

Reply:
Fixed